apps/next/src/pages/AccountManagement/components/Wallets/components/BulkDeleteButtons.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Button, Slide, Stack } from '@avalabs/k2-alpine';
+import { useAccountManager } from '@core/ui';
+import { FC, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+
+export const BulkDeleteButtons: FC = () => {
+  const { t } = useTranslation();
+  const { isManageMode, toggleManageMode, exitManageMode, selectedAccounts } =
+    useAccountManager();
+  const [showButtons, setShowButtons] = useState(isManageMode);
+  const { push } = useHistory();
+
+  return (
+    <>
+      {!isManageMode && (
+        <Box mx="auto">
+          <Button
+            variant="contained"
+            size="small"
+            color="secondary"
+            onClick={() => {
+              toggleManageMode();
+              setShowButtons(true);
+            }}
+          >
+            {t('Manage')}
+          </Button>
+        </Box>
+      )}
+      <Slide
+        direction="up"
+        in={isManageMode && showButtons}
+        mountOnEnter
+        unmountOnExit
+        onExited={() => exitManageMode()}
+      >
+        <Stack direction="column" gap={1} marginTop="auto">
+          <Button
+            variant="contained"
+            color="primary"
+            size="small"
+            fullWidth
+            disabled={selectedAccounts.length === 0}
+            onClick={() =>
+              push({
+                pathname: '/account-management/delete-account',
+                search: new URLSearchParams(
+                  selectedAccounts.map((id) => ['accountId', id]),
+                ).toString(),
+              })
+            }
+          >
+            {t('Delete selected')}
+          </Button>{' '}
```

Comment:
Is the `{' '}` needed? (To be honest I tried what happens if I delete it and I didn't see any difference, but I might have missed something)
\#justAsking